src/main/java/com/conveyal/gtfs/loader/EntityPopulator.java

```diff
@@ -214,9 +243,65 @@
         stopTime.continuous_drop_off = getIntIfPresent   (result, "continuous_drop_off", columnForName);
         stopTime.timepoint           = getIntIfPresent   (result, "timepoint", columnForName);
         stopTime.shape_dist_traveled = getDoubleIfPresent(result, "shape_dist_traveled", columnForName);
+        stopTime.pickup_booking_rule_id = getStringIfPresent(result, "pickup_booking_rule_id", columnForName);
+        stopTime.drop_off_booking_rule_id = getStringIfPresent(result, "drop_off_booking_rule_id", columnForName);
+        stopTime.start_pickup_dropoff_window = getIntIfPresent(result, "start_pickup_dropoff_window", columnForName);
+        stopTime.end_pickup_dropoff_window = getIntIfPresent(result, "end_pickup_dropoff_window", columnForName);
```

Comment:
These should be strings per https://github.com/MobilityData/gtfs-flex/blob/master/spec/reference.md#stop_timestxt-file-extended.

Reply:
Same as above -- I believe we convert it to Int somewhere. Int was used here to conform with Stop which also uses Int for the TIME type

Reply:
A conversion takes place on loading using `Entity.getTimeField()`. I think it is correct to leave these fields as TIME/Int fields.

Reply:
Thanks for checking.